conf/application.conf

```diff
@@ -207,11 +207,5 @@ go-live-date = "2022-04-01"
 # Feature Flags
 # Generic Feature Flags
 features {
-  isPreLaunch = false
-  ukCompanyPrivateBeta = false
   partnershipEnabled = true
 }
-
-# User Specific Feature Flags
-# empty list means the allowlist is disabled
-allowedUsers = []
```

Comment:
dont forget base and envs configs when this is deployed :)

Reply:
Well noticed Pan. I will liaise with Paul about that. Thanks